queries/fortran/highlights.scm

```diff
@@ -34,7 +34,7 @@
 [
 "implicit"
 (none)
-] @annotation
+] @attribute
```

Comment:
there is still the PR to do `@attribute` -> `@annotation` for all queries that are not html attributes.